### ISSUE TITLE
Namespace is always "Default"

### ISSUE THREAD

**Issue by borisdaeppen** [Akron/Mojolicious-Plugin-CHI#2]
Hello,

I'm trying to make this plugin work since quite some hours (for perlybook.org) but I keep on failing.
No clue what I'm doing wrong or if this is a bug?

In the main App I do something like this:

``` perl
$self->plugin(CHI => {
  cpan2ebook => {
    driver     => 'File',
    root_dir   => 'home/bo/tmp',
  }
}); 
```

And in the Controller something like this:

``` perl
$self->chi('cpan2ebook')->set("KEY", "DATA");
```

Now when I do this, nothing gets created in "/home/bo/tmp".

Before the above case I tried a variant where I pass the ref to a sub.

``` perl
# abstract pseudo code
sub my_cache_wrapper_sub {
  my $cache = shift;

  $cache->set("KEY", "DATA");
}

my_cache_wrapper_sub( $self->chi('cpan2ebook') );
```

In this case it did create the cache, but as "Default" instead of "cpan2ebook".

So in short: I have not been able to achieve anything working as suspected.
Can anybody give me a hint out of my examples, or even try if it runs for him?
Is the module already capable of this stuff?
Maybe I have some weird typo somewhere but before I continue searching for hours it would be great if somebody can give some feedback on this, if possible.

Thanks a lot.
